### PR TITLE
fix(fa): Updating about form in osk

### PR DESCRIPTION
**Pull request by MargretFinnboga**
# Updating about form in osk

https://app.asana.com/0/1201301944561348/1201257203547398
https://app.asana.com/0/1201301944561348/1201097099198289

## What

Update about when applicant is applying 
Added spouse name to form

## Why

Design and web are in sync

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx [hunk 1]
```diff
@@ -82,6 +89,156 @@ const AboutFormApplicant = () => {
...
+      </Text>
+
+      <Accordion singleExpand>
+        <AccordionItem
+          id="id_1"
+          label="Nánar um persónuverndarstefnu þíns sveitarfélags"
+        >
+          <Text marginBottom={[2, 2, 4]}>
+            Til að geta sannreynt rétt umsækjanda skv. umsókn þessari, í samræmi
+            við reglur Hafnarfjarðarkaupstaðar og gildandi lög varðandi veitingu
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx:103]: The municipality name should not be hardcoded here, right?

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx [hunk 2]
```diff
@@ -82,6 +89,156 @@ const AboutFormApplicant = () => {
...
+            þjónustunnar, þá þarf fjölskyldu- og barnamálasvið
+            Hafnarfjarðarkaupstaðar að afla persónuupplýsinga um umsækjanda.
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx:105]: And also not here

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx [hunk 3]
```diff
@@ -82,6 +89,156 @@ const AboutFormApplicant = () => {
...
+            Verði umbeðnar nauðsynlegar upplýsingar ekki veittar kann það að
+            leiða til þess að ekki er unnt að verða við umsókn þinni.
+          </Text>
+
+          <Text marginBottom={[1, 1, 2]}>
+            Með því að senda inn umsókn þessa staðfestir umsækjandi að hann hafi
+            kynnt sér efni varðandi vinnslu persónuupplýsinga af hálfu
+            fjölskyldu- og barnamálasvið Hafnarfjarðarkaupstaðar vegna umsóknar
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx:113]: And here

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx [hunk 4]
```diff
@@ -82,6 +89,156 @@ const AboutFormApplicant = () => {
...
+            persónuupplýsinga".
+          </Text>
+
+          <Text marginBottom={[1, 1, 2]}>
+            Persónuvernd og vinnsla persónuupplýsinga
+          </Text>
+
+          <Text marginBottom={[1, 1, 2]}>
+            Svo unnt sé að vinna úr umsókn þinni og leggja mat á rétt þinn til
+            þjónustu hjá Hafnarfjarðarkaupstað samkvæmt umsókninni, þá þarf
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx:124]: 😅

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx [hunk 5]
```diff
@@ -82,6 +89,156 @@ const AboutFormApplicant = () => {
...
+            fjölskyldu- og barnamálasvið Hafnarfjarðarkaupstaðar að vinna með
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx:125]: 😱

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx [hunk 6]
```diff
@@ -82,6 +89,156 @@ const AboutFormApplicant = () => {
...
+            nánar tilgreindar persónuupplýsingar þínar. Unnið er með þær
+            upplýsingar sem þú veitir í umsóknarferlinu og jafnframt þær
+            upplýsingar sem nauðsynlegar eru og tilgreindar eru hér til að
+            sannreyna að skilyrði reglna Hafnarfjarðarkaupstaðar séu uppfyllt
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx:129]: 🙀

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx [hunk 7]
```diff
@@ -82,6 +89,156 @@ const AboutFormApplicant = () => {
...
+            við upphaf og á meðan þjónustusamband aðila stendur yfir.
+          </Text>
+
+          <Text marginBottom={[1, 1, 2]}>
+            Tilgangur vinnslu og lagagrundvöllur Persónuupplýsingar þær sem
+            óskað er eftir á umsóknarformi þessu og unnið er með, eru
+            nauðsynlegar fjölskyldusviði Hafnarfjarðarkaupstaðar til að geta
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx:136]: 😢

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx [hunk 8]
```diff
@@ -82,6 +89,156 @@ const AboutFormApplicant = () => {
...
+            sannreynt og tryggt rétt umsækjanda til þjónustu í samræmi við
+            umsókn þessa, skv. eftirfarandi lögum: <br /> V, VI. og VII kafli
+            laga um félagsþjónustu sveitarfélaga nr. 40/1991.
+          </Text>
+          <Text marginBottom={[2, 2, 4]}>
+            Reglur Hafnarfjarðarkaupstaðar um fjárhagsaðstoð hjá
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx:142]: 😭

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx [hunk 9]
```diff
@@ -82,6 +89,156 @@ const AboutFormApplicant = () => {
...
+            Hafnarfjarðarbæ, síðast breytt 15. mars 2020.
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx:143]: :O

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx [hunk 10]
```diff
@@ -82,6 +89,156 @@ const AboutFormApplicant = () => {
...
+            húsnæðisstaða, atvinnustaða, félagsleg staða og aðstaða,
+            þjónustuþörf og aðrar upplýsingar sem umsækjandi vill koma á
+            framfæri í umsóknarferli. Viðkvæmar persónuupplýsingar:
+            heilsufarsupplýsingar.
+          </Text>
+
+          <Text marginBottom={[1, 1, 2]}>
+            Hvaðan koma upplýsingarnar? Umsækjandi veitir þær upplýsingar sem
+            unnið er með vegna framkominnar umsóknar auk þess sem
+            Hafnarfjarðarkaupstaður sækir grunnupplýsingar um umsækjanda til
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx:160]: ❌

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx [hunk 11]
```diff
@@ -82,6 +89,156 @@ const AboutFormApplicant = () => {
...
+            Tryggingastofnun ríkisins, Hagstofa Íslands,
+            atvinnuleysistryggingasjóði og útlendingastofnun.{' '}
+          </Text>
+
+          <Text marginBottom={[2, 2, 4]}>
+            Hversu lengi eru persónuupplýsingarnar varðveittar:
+            Persónuupplýsingar þínar eru geymdar ótímabundið á grundvelli
+            lagaskyldu sem hvílir á Hafnarfjarðarkaupstað skv. lögum um opinber
+            skjalasöfn nr. 77/2014. Almennt að liðnum 30 árum er
+            Hafnarfjarðarkaupstað skylt að afhenda Þjóðskjalasafni til
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx:182]: 👎

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx [hunk 12]
```diff
@@ -82,6 +89,156 @@ const AboutFormApplicant = () => {
...
+            varðveislu þau skjöl sem myndast við umsókn þína.
+          </Text>
+
+          <Text marginBottom={[2, 2, 4]}>
+            Öryggi persónuupplýsinganna: Hafnarfjarðarkaupstaður gætir öryggis
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx:187]: ⛔

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx [hunk 13]
```diff
@@ -82,6 +89,156 @@ const AboutFormApplicant = () => {
...
+            persónuupplýsinga með viðeigandi skipulagslegum og tæknilegum
+            ráðstöfunum, þ. á m. aðgangsstýringum og dulkóðun. Allt það
+            starfsfólk sem kemur að vinnslu persónuupplýsinga umsækjanda er
+            bundið þagnarskyldu.
+          </Text>
+
+          <Text marginBottom={[2, 2, 4]}>
+            Miðlun persónuupplýsinganna til þriðja aðila Tilteknum upplýsingum
+            sem Hafnarfjarðarkaupstaður hefur aflað í tengslum við umsókn þessa
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx:196]: ✖️

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx [hunk 14]
```diff
@@ -82,6 +89,156 @@ const AboutFormApplicant = () => {
...
+            er miðlað til þriðja aðila. Aðeins er miðlað þeim upplýsingum sem
+            eru nauðsynlegar til að veita hér umbeðna þjónustu.
+            Persónuupplýsingar eru ekki sendar annað án leyfis umsækjanda. Nánar
+            tiltekið er um að ræða:
+          </Text>
+
+          <Text marginBottom={[2, 2, 4]}>
+            Réttindi: Þú kannt að eiga rétt til þess að fá aðgang að þeim
+            persónuupplýsingum sem Hafnarfjarðarkaupstaður vinnur með í tengslum
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx:205]: 🛑

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx [hunk 15]
```diff
@@ -82,6 +89,156 @@ const AboutFormApplicant = () => {
...
+            við umsókn þína. Þá kannt þú að hafa rétt til að andmæla vinnslunni,
+            fá upplýsingar leiðréttar, krefjast þess að þeim verði eytt eða fara
+            fram á að vinnslan verði takmörkuð.
+          </Text>
+
+          <Text marginBottom={[2, 2, 4]}>
+            Kvörtun yfir vinnslu persónuupplýsinga: Sérstök athygli er vakin á
+            því að ef þú ert ósátt/ur við meðferð Hafnarfjarðarkaupstaðar á
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx:213]: 🙅

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx [hunk 16]
```diff
@@ -82,6 +89,156 @@ const AboutFormApplicant = () => {
...
+            persónuupplýsingum þínum getur þú ávallt haft samband við
+            persónuverndarfulltrúa Hafnarfjarðarkaupstaðar{' '}
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx:215]: 🚳

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx [hunk 17]
```diff
@@ -82,6 +89,156 @@ const AboutFormApplicant = () => {
...
+            <a
+              href={`mailto:personuvernd@hafnarfjordur.is`}
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx:217]: 🔕

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx [hunk 18]
```diff
@@ -82,6 +89,156 @@ const AboutFormApplicant = () => {
...
+              rel="noreferrer noopener"
+            >
+              <span className="linkInText">
+                {' '}
+                (personuvernd@hafnarfjordur.is)
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx:222]: 🚫

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx [hunk 19]
```diff
@@ -82,6 +89,156 @@ const AboutFormApplicant = () => {
...
+              </span>
+            </a>{' '}
+            eða sent erindi til Persónuverndar.
+          </Text>
+
+          <Text marginBottom={1}>
+            Frekari upplýsingar um vinnslu persónuupplýsinga hjá
+            Hafnarfjarðarkaupstað má finna í persónuverndarstefnu
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx:230]: :P

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormSpouse.tsx [hunk 20]
```diff
@@ -1,11 +1,15 @@
 import React, { useContext } from 'react'
 import { Text } from '@island.is/island-ui/core'
 
-import { months, nextMonth } from '@island.is/financial-aid/shared/lib'
+import {
+  currentMonth,
+  months,
+  nextMonth,
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormSpouse.tsx:7]: Are we still using next month here?

**Review by Rikkiabb — CHANGES_REQUESTED** [review]
🚳

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormSpouse.tsx (hunk 20 above)
Comment (MargretFinnboga) [reply apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormSpouse.tsx:7]: Yass below 👇

**Review by Rikkiabb — COMMENTED** [review]
One character ⚪

File: apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx [hunk 21]
```diff
@@ -82,6 +89,150 @@ const AboutFormApplicant = () => {
...
+            Tilgangur vinnslu og lagagrundvöllur Persónuupplýsingar þær sem
+            óskað er eftir á umsóknarformi þessu og unnið er með, eru
+            nauðsynlegar fjölskyldusviði {municipality?.name} til að geta
+            sannreynt og tryggt rétt umsækjanda til þjónustu í samræmi við
+            umsókn þessa, skv. eftirfarandi lögum: <br /> V, VI. og VII kafli
+            laga um félagsþjónustu sveitarfélaga nr. 40/1991.
+          </Text>
+          <Text marginBottom={[2, 2, 4]}>
+            Reglur {municipality?.name} um fjárhagsaðstoð hjá{' '}
+            {municipality?.name} , síðast breytt 15. mars 2020.
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/routes/application/AboutForm/aboutFormApplicant.tsx:143]: Extra space here

**Review by Rikkiabb — APPROVED** [review]

**Pull request merged**